tests/test_client.py

```diff
@@ -901,9 +901,9 @@ def test_recursive_authorize(self):
         token_response = {'access_token': '1/3w', 'expires_in': 3600}
         encoded_response = json.dumps(token_response).encode('utf-8')
         http = http_mock.HttpMock(data=encoded_response)
-        http = self.credentials.authorize(http)
-        http = self.credentials.authorize(http)
-        transport.request(http, 'http://example.com')
+        authed_http = self.credentials.authorize(http)
+        authed_http = self.credentials.authorize(http)
```

Comment:
Haha, the intent of the original code is to double-wrap the object, but I messed that up. Fixed.